include/RED4ext/Scripting/Natives/entAnimationControllerComponent.hpp

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+// clang-format off
+
+// This file is generated from the Game's Reflection data
+
+#include <RED4ext/Common.hpp>
+#include <RED4ext/Handle.hpp>
+#include <RED4ext/NativeTypes.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/AnimDatabaseCollection.hpp>
+#include <RED4ext/Scripting/Natives/Generated/ent/IComponent.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/IKTargetRef.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/IKTargetParams_Update.hpp>
+#include <RED4ext/Scripting/Natives/Generated/ent/IOrientationProvider.hpp>
+#include <RED4ext/Scripting/Natives/Generated/ent/IPositionProvider.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/LookAtRef.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/LookAtParams_UpdatePositions.hpp>
+#include <cstdint>
+
+namespace RED4ext
+{
+namespace anim { struct ActionAnimDatabase; }
+namespace ent
+{
+struct AnimationControlBinding;
+struct AnimationControllerComponent;
+
+struct IKTargetController
+{
+    struct IKTargetData
+    {
+        anim::IKTargetRef ikTargetref;                    // 00
+        Handle<IPositionProvider> positionProvider;       // 10
+        Handle<IOrientationProvider> orientationProvider; // 20
+    };
+    AnimationControllerComponent& animationControllerComponent; // 00
+    uint64_t unk08;                                             // 08
```

Comment:
Maybe it's `Handle<AnimationControllerComponent>`? Same for `LookAtController`.

Reply:
It's just a pointer in memory, so I left it as a reference. Not sure Handle would fit here